packages/contracts-bedrock/test/L1/OPContractsManager.t.sol

```diff
@@ -444,6 +444,9 @@ contract OPContractsManager_Upgrade_Harness is CommonTest {
         assertEq(ISemver(address(pdg)).version(), "1.4.1");
         assertEq(address(pdg.vm()), impls.mipsImpl);
 
+        // Check that the SystemConfig is upgraded to the expected version
+        assertEq(ISemver(address(systemConfig)).version(), "2.5.0");
```

Comment:
also check the implementation address matches what's in the opcm - you can use the EIP1967 helper to do that.

Reply:
Done